### PR TITLE
a3_stereo_image_proc benchmark

### PR DESCRIPTION
**Pull request by jasonjabbour**
the a3_stereo_image_proc benchmark is ready for review! Other than the data analysis, I believe it is fully complete. I would love to get some feedback. I will give a brief demo to the working group tomorrow!

### CODE REVIEW

**Comment by vmayoral** [comment]
THanks @jasonjabbour! Couple of aspects that catched my eye quickly, you're including lots of file resulting from the build and install actions, these should not be in the source code. Can you clean that up so that reviewing the source code becomes simplified? If you do so and have a demo for later today, I believe we should be able to get this in rather quickly!

File: benchmarks/perception/install/.colcon_install_layout
Comment (vmayoral) [inline benchmarks/perception/install/.colcon_install_layout:1]: This file and all stuff under `install` should be removed.

File: benchmarks/perception/log/build_2023-03-15_04-17-47/a3_stereo_image_proc/command.log
Comment (vmayoral) [inline benchmarks/perception/log/build_2023-03-15_04-17-47/a3_stereo_image_proc/command.log:1]: same with the `log` folder

Comment (jasonjabbour) [reply benchmarks/perception/log/build_2023-03-15_04-17-47/a3_stereo_image_proc/command.log:1]: Thanks Victor! also the build folder?

Comment (jasonjabbour) [reply benchmarks/perception/log/build_2023-03-15_04-17-47/a3_stereo_image_proc/command.log:1]: oh nvm got it!

**Pull request closed without merging**